### DISCUSSION TITLE
Frequently Asked Questions | Chulapa

### DISCUSSION THREAD

**Discussion by giscus** [dieghernan/chulapa#24] [General]
# Frequently Asked Questions | Chulapa

Some additional information you may find useful - Advices and FAQs

https://dieghernan.github.io/chulapa/docs/05-faq

**Comment by dieghernan** [comment]
Cool tricks, do you know any more?

**Comment by PaleSkinnySwede** [comment]
Just came across this theme as I was browsing dark Jekyll themes. Is it possible to change the background of the code snippet to black? It's a bit weird that it's white in all the dark Chulapa themes.

**Reply by dieghernan** [reply]
Hi! We provide 70~ syntax highlighting themes, most of them dark. Check https://dieghernan.github.io/chulapa/docs/syntax-highlighting

And configure it in the corresponding section of the _config.yaml.

**Reply by dieghernan** [reply]
See https://dieghernan.github.io/celestial_data/ and the source code in https://github.com/dieghernan/celestial_data/blob/ecd4eebfed1433b7c21ff5b9bae7fcedc649e100/_config.yml#L81

**Reply by PaleSkinnySwede** [reply]
> Hi! We provide 70~ syntax highlighting themes, most of them dark. Check https://dieghernan.github.io/chulapa/docs/syntax-highlighting
> 
> And configure it in the corresponding section of the _config.yaml.

Sweet. I'll go though the settings to see what I can change. I really like the "Butterfly Mid" skin.

**Comment by joanllobera** [comment]
Hi,
this is a nice theme. Thanks for making this available broadly.

 I am not sure if this is a chulapa-specific question, or a general jekyll question. I hope you can orient me in the right direction.

I am trying to set up a page that shows posts for specific categories or tags. I am not managing to figure out how to tell a page "show the blog posts that correspond to tag X or category Y"

To illustrate my point, here are some specifics:

 I have modified the chulapa template and added a blog page that shows 3 blog posts: https://joanllobera.github.io/labreuvoir/blog/

I would like to have a page that only shows the blog posts that correspond to the category "music", or which contain the tag "musique", whichever is simpler. It should be here: 
https://joanllobera.github.io/labreuvoir/music 

when defining this page, I have put the appropriate label in  "include_collection", but it does not seem to be working
(see what I did here: https://github.com/joanllobera/labreuvoir/blob/main/_pages/music.md )

I am wondering if someone can give me a pointer on how to filter based on content tags or based on content categories.

**Reply by dieghernan** [reply]
See this to show only the collection `projects`

Source: https://github.com/dieghernan/dieghernan.github.io/blob/main/_pages/projects.md

Render: https://dieghernan.github.io/projects

More docs: https://dieghernan.github.io/chulapa/docs/04-layouts#index-category

**Reply by dieghernan** [reply]
You may need to set a Jekyll collection:

[collections_dir     : collections](https://github.com/dieghernan/dieghernan.github.io/blob/b8950c1c37d74edafe3ceff3e568545ba617dcb9/_config.yml#L138)

https://jekyllrb.com/docs/step-by-step/09-collections/


https://jekyllrb.com/docs/collections/
